### ISSUE TITLE
Airlock review VM creation requires SAS URL, but request does not provide one

### ISSUE THREAD

**Issue by TonyWildish-BH** [microsoft/AzureTRE#3913]
**Describe the bug**
I have successfully made a few airlock import requests, and can see them in the TRE interface.

When I try to create a review VM for them, they require a SAS token to be able to access the data.

There seems to be no way to generate this SAS URL. The documentation shows a screen shot which has a 'generate' button (https://microsoft.github.io/AzureTRE/latest/using-tre/tre-for-research/review-airlock-request/), but the TRE I have deployed does not have that button (see screenshot).

**Steps to reproduce**

1. Create an airlock import request,
1. Attempt to create an airlock review VM,
1. Fail, because you can't get a SAS URL to use to create the VM.

For bonus points, if you leave the SAS URL empty (it's optional in the interface), you don't get a complaint until it starts to deploy the service and fails several minutes later.

**Azure TRE release version (e.g. v0.14.0 or main):**
main

**Deployed Azure TRE components - click the (i) in the UI:**
UI Version: 0.5.21
API Version: 0.18.5

<img width="444" alt="Screenshot 2024-04-26 at 5 12 25 PM" src="https://github.com/microsoft/AzureTRE/assets/153200306/50cb52f3-b812-44ba-ab94-765ac5289e96">

**Comment by tim-allen-ck** [comment]
Hi @TonyWildish-BH very odd, what permissions does your user have in that workspace?

**Comment by TonyWildish-BH** [comment]
Hi @tim-allen-ck. I have all roles; Owner, Research, and Airlock Manager

**Comment by tim-allen-ck** [comment]
Does the VM create?

**Comment by TonyWildish-BH** [comment]
No, it fails early with `Error message: parameter "airlock_request_sas_url" is required`, at the `porter install` step.

**Comment by tim-allen-ck** [comment]
I think that screenshot in the docs is out of date, I'm not sure that generate button doesn't exist to my knowledge. 
I'm assuming the airlock workspace is configured correctly too?

**Comment by TonyWildish-BH** [comment]
I don't have an airlock workspace, I'm creating an import review VM in Guacamole in the workspace I created the import request in.

**Comment by akolensky** [comment]
Hi Tony, this link should help:
https://microsoft.github.io/AzureTRE/latest/tre-admins/configure-airlock-review/

I believe the documentation link in the post is oudated, but the one above should help.
Following that process, it becomes possible to create Review VMs 
![image](https://github.com/microsoft/AzureTRE/assets/125355163/dd7a7812-bdfa-4b6f-868a-fd45aa5c8c24)

and then validate the data in the ReviewData folder
![image](https://github.com/microsoft/AzureTRE/assets/125355163/8ed741d9-7de4-4837-8a16-a6c0757a63b0)

**Comment by tim-allen-ck** [comment]
@TonyWildish-BH did the docs above help? can I close this issue?

**Comment by TonyWildish-BH** [comment]
hi @tim-allen-ck, I've been off for a while, and haven't had the chance to follow up on this yet, so please don't close it yet.

**Comment by marrobi** [comment]
@TonyWildish-BH I trust this can be closed, no update for 6 months?
